src/main/java/core/basesyntax/model/Transaction.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.model;
+
+public class Transaction {
+    private final String operation;
+    private final String fruitName;
+    private final int quantity;
```

Comment:
Consider making the fields `operation`, `fruitName`, and `quantity` private to follow the encapsulation principle. Currently, they are package-private, which is more permissive than necessary for this use case.